pkg/utils/constant.go

```diff
@@ -0,0 +1,8 @@
+package utils
+
+const (
+	PortNo    = ":8080"
+	HMDB      = "HMDB"
```

Comment:
Also change the db name, HMDB is not appropriate. Suggestion: `hospital-management`